cmd/geth/verkle.go

```diff
@@ -75,7 +75,7 @@ func checkChildren(root verkle.VerkleNode, resolver verkle.NodeResolverFn) error
 	switch node := root.(type) {
```

Comment:
Multiple code changes are related to:
- Pulling `master` from `go-verkle`, since [my other PR](https://github.com/gballet/go-verkle/pull/300) is sitting on top of it.
- Linting/Formatting.